src/main/java/com/kodilla/ecommercee/ProductOrder.java

```diff
@@ -0,0 +1,27 @@
+package com.kodilla.ecommercee;
+
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+@Entity(name = "productOrders")
```

Comment:
Lepiej nie nadpisywać nazwy w Entity, jeśli zostawisz samo: 

` @Entity`

To wystarczy